### DISCUSSION TITLE
No Challenges = Error?

### DISCUSSION THREAD

**Discussion by Roseghost** [alkem-io/alkemio#449] [Ideas] [answered]
Currently you get an error message when you don't have challenges. However theres nothing going wrong. Maybe change it to a placeholder text that says ''The are no challenges yet'' 

No the user thinks theres something wrong and they might try to reload their page till they do see challenges. 

![image](https://user-images.githubusercontent.com/64191883/137275105-8414e2a6-dd5a-400f-82b4-8c75f8341256.png)

**Comment by Comoque1** [comment, marked as answer]
Hi Rose, thank you for the feedback.
The [issue](https://app.zenhub.com/workspaces/alkemio-5ecb98b262ebd9f4aec4194c/issues/alkem-io/client-web/1245) is tracked and will be addressed with some of the following releases.
